src/main/java/seedu/address/ui/TaskListCard.java

```diff
@@ -18,6 +18,8 @@ public class TaskListCard extends UiPart<Region> {
     @FXML
     private VBox cardPane;
     @FXML
+    private VBox taskCard;
```

Comment:
This refers to the `VBox` in the `TaskListCard` which can be shown and hidden.

I will rename this variable in the next pull request.